app/src/main/java/com/base_android_template/usecase/GetGithubUsersUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.base_android_template.usecase
+
+import com.base_android_template.model.response.GithubUserResponse
+import com.base_android_template.repository.GithubUsersRepository
+import com.base_android_template.shared.network.ApiResponse
+
+class GetGithubUsersUseCase internal constructor(
+    private val githubUsersRepository: GithubUsersRepository
+) {
```

Comment:
@LoredanaZdranc it would be better for this to implement an interface. That would allow us to change this behavior in tests